mix.exs

```diff
@@ -55,7 +55,11 @@ defmodule Appsignal.Phoenix.MixProject do
       end
 
     [
-      {:appsignal_plug, ">= 2.0.8 and < 3.0.0"},
+      {
+        :appsignal_plug,
+        ">= 2.0.8 and < 3.0.0",
+        github: "appsignal/appsignal-elixir-plug", branch: "replace-set-sample-data"
```

Comment:
I think the future it's okay not to commit this and have the build fail until the PR this is blocked on is merged. Reduces the amount of rebasing and force pushing.

Reply:
Noted! I mostly do it for my own peace of mind, though. Seeing the green tick in the CI check soothes me.